src/Runtime/Optimized/Physics/mmdBulletPhysics.ts

```
@@ -0,0 +1,640 @@
+                break;
+
+            case PmxObject.RigidBody.PhysicsMode.Physics:
+            case PmxObject.RigidBody.PhysicsMode.PhysicsWithBone:
+                break;
+
+            default:
+                throw new Error(`Unknown physics mode: ${data.physicsMode}`);
+            }
+        }
+    }
+    
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)